docs/portal/client-side-integration.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Client-Side Integration
+description: Set up and manage information needed for client-side integration.
+hide_table_of_contents: false
+sidebar_position: 09
+---
+
+# Client-Side Integration
+
+In the UID2 Portal, if you want to use an implementation option that generates UID2 tokens on the client side, you'll need to define one or more of each of the values shown in the following table.
+
+| Value | Details | Documentation Link |
+| :--- | :--- | :---|
+| Key pair | At least one. In your implementation, you'll share the public key. | [Subscription ID and Public Key](getting-started/gs-credentials.md#subscription-id-and-public-key) |
+| Root-level domain | At least one. Provide a complete list of your domains. | [Client-Side Implementation for Publishers](../getting-started/gs-account-setup.md#client-side-implementation-for-publishers) |
```

Comment:
We are calling it "top-level domain" everywhere in the portal, they might mean the same thing but probably best to keep it consistent

Reply:
I see top-level domain and root-level domain are used in the docs, but I'd say stick with top-level domain to keep consistent

Reply:
Done, thx